.github/workflows/t3000-unit-tests.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           { name: "t3k ttnn tests", arch: wormhole_b0, cmd: run_t3000_ttnn_tests, timeout: 120, owner_id: UAFM0F6FM}, #Akhmed Rakhmati
           { name: "t3k falcon7b tests", arch: wormhole_b0, cmd: run_t3000_falcon7b_tests, timeout: 30, owner_id: UBHPP2NDP}, #Joseph Chu
           { name: "t3k falcon40b tests", arch: wormhole_b0, cmd: run_t3000_falcon40b_tests, timeout: 30, owner_id: U044T8U8DEF}, #Johanna Rock
-          { name: "t3k mixtral tests", arch: wormhole_b0, cmd: run_t3000_mixtral_tests, timeout: 30, owner_id: U03PUAKE719}, #Miguel Tairum Cruz
+          { name: "t3k mixtral tests", arch: wormhole_b0, cmd: run_t3000_mixtral_tests, timeout: 45, owner_id: U03PUAKE719}, #Miguel Tairum Cruz
```

Comment:
why are we extending the time for mixtral unit tests here?

Reply:
@ttmchiou the new Mixtral prefill unit tests take longer to complete since we run them for 7 different prefill sequence lengths. When I measure the time it would take to run all the tests it was a bit over 30 min if I recall correctly. 

We can try to reduce the number of tests to keep it under 30 min if this is a problem for CI.

Reply:
If it's a 30 minute long test, would it be better suited to run on the t3k frequent pipeline?
Generally tests run on unit tests should be fast and average around 15 minutes at most. the 30 minute timeout is for for catching bugs such as if the machine or code hangs as a safety net. 

Bumping up the timeout should usually be a last resort since the pipeline itself would become lopsided if we have 1 test running 30 minutes while the others are running around 15 minutes.

That's not to say we want to cut down on code quality or the reduce code coverage but usually significantly increasing timeouts is a sign that something may be suited to other types of pipeliens

![image](https://github.com/user-attachments/assets/56bf4b68-fadd-419f-aa8c-342db9ba810a)

Reply:
That makes sense. I've cleaned up some of the prefill tests to reduce timing and moved the longer ones from unit to frequent pipeline. 

And restored the original 30 min timeout for unit tests 👍 

Re-running pipelines and will merge if they all pass.